examples/speaker_recognition/notebooks/Speaker_Recognition_an4.ipynb

```diff
@@ -0,0 +1,1998 @@
+            "/content/data/an4/wav/an4_clstk/fsrb/an170-fsrb-b.wav\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "cN09z0XFoDjN",
+        "colab_type": "text"
+      },
+      "source": [
+        "Since we created scp file for train, we use `scp_to_manifest.py` to convert this scp file to manifest and then optionally split the files to train \\& dev for evaluating the models while training by using `--split` flag. So as you guessed we wouldn't be needing `--split` option for test folder. "
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "fNXZwNexIkAo",
+        "colab_type": "code",
+        "outputId": "ca06c4be-c0f6-4ec7-8198-a26347ea4b1e",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 85
+        }
+      },
+      "source": [
+        "!python scp_to_manifest.py --scp data/an4/wav/an4_clstk/train_all.scp --id -2 --out data/an4/wav/an4_clstk/all_manifest.json --split"
+      ],
+      "execution_count": 5,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "100% 948/948 [01:16<00:00, 12.39it/s]\n",
+            "853\n",
+            "wrote data/an4/wav/an4_clstk/train.json\n",
+            "wrote data/an4/wav/an4_clstk/dev.json\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "dxUL_g77oned",
+        "colab_type": "text"
+      },
+      "source": [
+        "Generating scp for test folder and then converting to manifest type. "
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "QShlVwEIO64D",
+        "colab_type": "code",
+        "outputId": "291d7dce-e202-4062-9eee-e43224084cb5",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 34
+        }
+      },
+      "source": [
+        "!find $PWD/data/an4/wav/an4test_clstk  -iname \"*.wav\" > data/an4/wav/an4test_clstk/test_all.scp\n",
+        "!python scp_to_manifest.py --scp data/an4/wav/an4test_clstk/test_all.scp --id -2 --out data/an4/wav/an4test_clstk/test.json"
+      ],
+      "execution_count": 6,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "text": [
+            "100% 130/130 [00:10<00:00, 12.38it/s]\n"
+          ],
+          "name": "stdout"
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "SqYvWJd_TuhT",
+        "colab_type": "code",
+        "colab": {
+          "base_uri": "https://localhost:8080/",
+          "height": 1000
+        },
+        "outputId": "1dd8e60b-6cf0-4e20-ae19-374441cfb58c"
+      },
+      "source": [
+        "!git clone https://github.com/NVIDIA/NeMo.git\n",
+        "os.chdir('NeMo')\n",
+        "!bash reinstall.sh"
```

Comment:
If you are going to clone anyways, there is no point of the installation instructions in the first code block.

Reply:
Removing the first block, but later when pip package comes with speaker-related files, this should be removed

Reply:
can you move the clone and reinstall to the first block?

Reply:
then you can remove the wgets

Reply:
Done